website/docs/cdktf/community.html.md

```diff
@@ -7,10 +7,16 @@ description: "Resources to help you contribute to the codebase, submit issues, a
 
 # Community
 
-CDK for Terraform is an early experimental project and the development team would love your feedback to help guide the project.
+CDK for Terraform is a rapidly-growing project and the development team would love your feedback and direct contributions to help guide the project.
```

Comment:
```suggestion
CDK for Terraform is growing rapidly, and the development team would love your feedback and direct contributions to help guide the project.
```

Reply:
Rather than "growing rapidly" or "early experimental" maybe "CDK for Terraform is under active development" is a better way to phrase?

Reply:
okay! updated suggestion to reflect!